src/node_report_utils.cc

```diff
@@ -28,32 +26,23 @@ void ReportEndpoints(uv_handle_t* h, std::ostringstream& out) {
       break;
   }
   if (rc == 0) {
-    // getnameinfo will format host and port and handle IPv4/IPv6.
-    rc = getnameinfo(addr,
-                     addr_size,
-                     hostbuf,
-                     sizeof(hostbuf),
-                     portbuf,
-                     sizeof(portbuf),
-                     NI_NUMERICSERV);
-    if (rc == 0) {
-      out << std::string(hostbuf) << ":" << std::string(portbuf);
-    }
+    // uv_getnameinfo will format host and port and handle IPv4/IPv6.
+    uv_getnameinfo_t local;
+    rc = uv_getnameinfo(h->loop, &local, nullptr, addr, NI_NUMERICSERV);
+
+    if (rc == 0)
+      out << std::string(local.host) << ":" << std::string(local.service);
```

Comment:
I think you may be able to drop the `std::string()` conversions?

Reply:
Yep, the compiler seems fine with that. Thanks!